compiler/damlc/lib/DA/Cli/Damlc.hs

```diff
@@ -447,22 +447,27 @@ execIde telemetry (Debug debug) enableScenarioService options =
             -- performance will be significatly impacted by large log output.
             threshold
             "LanguageServer"
+          damlCacheM <- lookupEnv "DAML_CACHE"
```

Comment:
can we use the environment variable constants we defined in daml-project-config?

Reply:
changed.